src/app/member/page.tsx

```diff
@@ -1,9 +1,14 @@
 import React from "react";
 import { MemberList } from "./components/MemberList";
+import { MemeberHeader } from "./components/MemeberHeader";
+
 const Page = () => {
   return (
-    <div className="p-8">
-      <MemberList />
+    <div>
```

Comment:
Can we use React Fragment instead of div

Reply:
@Mubashar1009  Nope for styling we can’t use `React Fragment`.